src/open_inwoner/templates/pages/cases/status_inner.html

```diff
@@ -46,7 +46,7 @@ <h2 class="h2" id="statuses">{% trans 'Status' %}</h2>
 
             {#  Documents. #}
             {% if case.documents %}
-                <h2 class="h2" id="documents">{% trans 'Documenten' %}</h2>
+                <h2 class="h2" id="documents">{% trans 'Documenten' %}{% if case.new_docs %}{% icon icon="fiber_manual_record" outlined=False extra_classes="file__file--recent" %}{% endif %}</h2>
```

Comment:
I would maybe not use an icon here (because it takes space) - for example: I made a CSS circle for the notifications in our menu so it could be positioned absolutely- but for Absolute positioning you need a parent with Position:relative anyway.
So: look at the design and think of changing the structure to help you position the dot Absolute 🙂 or use a CSS border-radius circle if that makes it easier: https://www.figma.com/file/iKGhWhstaLIlFSaND2q7cE/OIP---Designs-(new)?type=design&node-id=8-15743&mode=design&t=CuHG1L2dDE2kGhX1-0

<img width="272" alt="example-notificationDot" src="https://github.com/maykinmedia/open-inwoner/assets/118177951/7f7fcf18-0b89-41fa-8cb7-93879db09db2">

Reply:
Oh ignore my CSS circle comment - if you use `vertical-align:middle` it should look more like in the design:

<img width="113" alt="Screenshot 2023-10-31 at 14 49 26" src="https://github.com/maykinmedia/open-inwoner/assets/118177951/d78bcdc0-0061-4bf6-a1fa-5df672341ea6">